sklearn/pipeline.py

```diff
@@ -1271,6 +1271,8 @@ def make_union(*transformers, n_jobs=None, verbose=False):
     Returns
     -------
     f : FeatureUnion
+        A ``FeatureUnion`` object for concatenating the results of multiple transformer
```

Comment:
This will create an hyperlink to the actual class.
```suggestion
        A :class:`FeatureUnion` object for concatenating the results of multiple
        transformer
```